pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs

```diff
@@ -0,0 +1,355 @@
+                    <input type="hidden" name="__action" value="upload_assessment_groups">
+                    <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+                    <button type="submit" class="btn btn-primary">Upload</button>
+                  </div>
+                </div>
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+    <div id="content" class="container-fluid">
+      <div class="modal fade" id="autoAssessmentGroupsModal" tabindex="-1" role="dialog" aria-labelledby="autoAssessmentGroups">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h4 class="modal-title" id="autoAssessmentGroupsModalLabel">Auto new group setting</h4>
+            </div>
+            <div class="modal-body">
+              <form name="auto-assessment-group-form" method="POST" enctype="multipart/form-data">
+                <div class="form-group">
+                  <label for="formMin">Min number of members in a group</label>
+                  <input type="text" class="form-control" name="minnum">
+                </div>
+                <div class="form-group">
+                  <label for="formMax">Max number of members in a group</label>
+                  <input type="text" class="form-control" name="maxnum">
+                </div>
+                <div class="radio">
+                  <label><input type="radio" name="optradio" value = 1 checked>Grouping entire class (Please delete all groups before executing)</label>
+                </div>
+                <div class="radio">
+                  <label><input type="radio" name="optradio" value = 2>Grouping not-assigned students (fill existing groups first)</label>
+                </div>
+                <div class="radio">
+                  <label><input type="radio" name="optradio" value = 3>Grouping not-assigned students (will not change existing groups; create new groups only)</label>
+                </div>
+                <div class="d-flex justify-content-end">
+                  <div class="form-group mb-0">
+                    <input type="hidden" name="__action" value="auto_assessment_groups">
+                    <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+                    <button type="submit" class="btn btn-primary">Group</button>
+                  </div>
+                </div>
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div id="content" class="container-fluid">
+        <div class="modal fade" id="copyAssessmentGroupsModal" tabindex="-1" role="dialog" aria-labelledby="copyAssessmentGroups">
```

Comment:
```suggestion
        <div class="modal fade" id="copyAssessmentGroupsModal" tabindex="-1" role="dialog" aria-labelledby="copyAssessmentGroupsModalLabel">
```
This should match the `id` on line 89. `aria-labelledby` tells screen readers where to find the title of this div.